tests/test_stimulus.py

```diff
@@ -167,3 +168,14 @@ def test_create_2d_bar_stimulus(direction: str):
             # All values in each column should be equal for all frames
             cols_equal = np.all(stimulus.design == stimulus.design[:, :, [0]])
             assert np.all(cols_equal)
+
```

Comment:
Should we also have a test for `make_video` itself?

Reply:
I added a hacky test that relies on internals of matplotlib's `animation`. Not sure there's a better way